docs/podman-container-runlabel.1.md

```diff
@@ -95,6 +95,23 @@ Require HTTPS and verify certificates when contacting registries (default: true)
 then tls verification will be used, If set to false then tls verification will not be used. If not specified
 tls verification will be used unless the target registry is listed as an insecure registry in registries.conf
 
+## Examples ##
+
+Execute the run label of an image called foobar.
+```
+$ sudo podman container runlabel run foobar
+```
+
+Execute the install label of an image called foobar with additional arguements.
```

Comment:
s/arguments/arguements

Reply:
It's the "run" label below but the comment states "install".

Reply:
fixed, ty